cmd/mattermost/commands/webhook.go

```diff
@@ -374,7 +375,9 @@ func modifyOutgoingWebhookCmdF(command *cobra.Command, args []string) error {
 		return fmt.Errorf("unable to find webhook '%s'", webhookArg)
 	}
 
-	updatedHook := model.OutgoingWebhookFromJson(strings.NewReader(oldHook.ToJson()))
+	var updatedHook model.OutgoingWebhook
+	js, _ := json.Marshal(oldHook)
```

Comment:
This seems like a place where we should handle the error?

Reply:
All these commands are getting removed so I opted not to make an effort.